vaadin-checkbox-flow-demo/src/main/java/com/vaadin/flow/component/checkbox/demo/CheckboxView.java

```diff
@@ -57,6 +64,80 @@ protected void initView() {
         configurationForRequired(); // Validation
         themeVariantsHorizontal();// Theme Variants
         styling(); // Styling
+        dataViewDemo();
+    }
+
+    private void dataViewDemo() {
```

Comment:
This is the demo view that is shown on vaadin.com component demos.

I think it contains many things that should be in a IT instead but not in a public facing demo.

Reply:
Yes, didn't know this is used publicly. Will move it to IT

Reply:
Two separated pages have been created - one for dataview and one for listdataview API. I propose create another PR for IT tests.